app/routes/apply/eligibility/claim/file-upload.js

```diff
@@ -139,12 +133,13 @@ function checkForMalware (req, res, next, redirectURL) {
   const ids = setReferenceIds(req)
   const claimId = addClaimIdIfNotBenefitDocument(req.query.document, req.session.claimId)
   if (req.file) {
-    clam.scan(req.file.path).then((infected) => {
+    clam.scan(req.file.path).then(async (infected) => {
```

Comment:
as someone who came to node after asyn/await I find all this callback stuff really difficult to reason about

does clam.scan return a promise that you could await on?

Reply:
Completely with you, promises are awful to think about and there are differences between the default behaviours of async/await and promises. This codebase is choc-full of promise-based stuff and I'm not looking forward to trying to sort it all out.